cns/configuration/configuration.go

```diff
@@ -30,7 +30,7 @@ type CNSConfig struct {
 	EnablePprof                 bool
 	EnableSubnetScarcity        bool
 	EnableSwiftV2               bool
-	SWIFTV2Mode                 SWIFTV2Mode
+	SWIFTV2Mode                 string
```

Comment:
I think that's what we are doing with `cniConflistScenario`? Since we would eventually pass this in from the config yaml, im not certain if Go could unmarshal a string into a custom type

Reply:
just to clarify, we still have those custom types, the string would be type casted [here](https://github.com/Azure/azure-container-networking/pull/2390/files#diff-3c6ba0ad51f93dc5c172cd00978b989c8c63d2b8d799f8d58d56c983b6498cceR1386)

Reply:
> im not certain if Go could unmarshal a string into a custom type

easy to test. the custom type would be preferable

Reply:
oh nice it works, idk why we are not doing that for `cniConflistScenario` though?